### DISCUSSION TITLE
can not start my projekt

### DISCUSSION THREAD

**Discussion by Leotrimbehrami** [tailwindlabs/tailwindcss#11478] [Help]
``I was working on my projekt and i choose to install tailwindcss in vite and make something with it, but I becomme an error and can not find the problem.
This is my `package.json`
`{
  "name": "my-portfolio",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
    "preview": "vite preview"
  },
  "dependencies": {
    "autoprefixer": "^10.4.14",
    "postcss": "^8.4.24",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scroll": "^1.8.9",
    "sass": "^1.63.4"
  },
  "devDependencies": {
    "@types/react": "^18.0.37",
    "@types/react-dom": "^18.0.11",
    "@vitejs/plugin-react": "^4.0.0",
    "@vitejs/plugin-react-refresh": "^1.3.6",
    "eslint": "^8.38.0",
    "eslint-plugin-react": "^7.32.2",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.3.4",
    "tailwindcss": "^3.3.2",
    "vite": "^4.3.9"
  }
}
`
tailwind.config.js
`export default {
  purge: [],
  darkMode: false, // oder 'media' oder 'class'
  theme: {
    extend: {},
  },
  variants: {
    extend: {},
  },
  plugins: [],
};
`
and vite.config.js
import { defineConfig } from 'vite';
import reactRefresh from '@vitejs/plugin-react-refresh';
import tailwindcss from 'tailwindcss';

import autoprefixer from 'autoprefixer';

```
export default defineConfig({
  plugins: [
    reactRefresh(),

    // Füge Tailwind CSS und Autoprefixer als Vite-Plugins hinzu
    tailwindcss(),
    autoprefixer(),
  ],
});
```
I know it is a lot of code but if someone can help me.
And here is the error
`error when starting dev server:
TypeError: optionsHook.call is not a function
    at file:///home/user/Desktop/My%20Portfolio/My%20Portfolio/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:42811:46
    at createPluginContainer (file:///home/user/Desktop/My%20Portfolio/My%20Portfolio/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:42821:11)
    at _createServer (file:///home/user/Desktop/My%20Portfolio/My%20Portfolio/node_modules/vite/dist/node/chunks/dep-e8f070e8.js:63294:29)
    at async CAC.<anonymous> (file:///home/user/Desktop/My%20Portfolio/My%20Portfolio/node_modules/vite/dist/node/cli.js:733:24)`

**Comment by Rafaelaxo** [comment]
Glad you like it. We have been using this for several weeks now and it seems to be going nicely. [MyAARPMedicare Sign In](https://www.myaarpmedicare.us/)

**Comment by wongjn** [comment]
It seems like [the installation instructions for Vite](https://tailwindcss.com/docs/guides/vite) were not followed.

Remove the PostCSS plugins being called in the `vite.config.js`:
```diff
 import { defineConfig } from 'vite';
 import reactRefresh from '@vitejs/plugin-react-refresh';
-import tailwindcss from 'tailwindcss';
-
-import autoprefixer from 'autoprefixer';

 export default defineConfig({
   plugins: [
     reactRefresh(),
-
-   // Füge Tailwind CSS und Autoprefixer als Vite-Plugins hinzu
-    tailwindcss(),
-    autoprefixer(),
   ],
 });
```
Add the PostCSS plugins in `postcss.config.js`:
```js
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
  },
};
```

**Reply by Leotrimbehrami** [reply]
ooh thank you very much. now is working
